systemverilog-plugin/UhdmAst.cc

```diff
@@ -1902,33 +1902,22 @@ void UhdmAst::process_module()
     } else {
         // Not a top module, create instance
         current_node = make_ast_node(AST::AST_CELL);
-        std::string module_parameters;
+        std::vector<std::pair<RTLIL::IdString, RTLIL::Const>> parameters;
         visit_one_to_many({vpiParamAssign}, obj_h, [&](AST::AstNode *node) {
             if (node && node->type == AST::AST_PARAMETER) {
                 if (node->children[0]->type != AST::AST_CONSTANT) {
```

Comment:
Shouldn't we assert `node->children.size() > 0` before accessing `children[0]` ?

Reply:
Right, added.